igor-web/src/main/groovy/com/netflix/spinnaker/igor/build/model/GenericArtifact.groovy

```diff
@@ -16,10 +16,17 @@
 
 package com.netflix.spinnaker.igor.build.model
 
+import com.fasterxml.jackson.annotation.JsonInclude
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
 class GenericArtifact {
     String fileName
     String displayPath
     String relativePath
```

Comment:
Im not sure, when we roll this out they will stay at least, when artifact decoration is used throughout spinnaker we can deprecate and remove them.